src/components/CurrentWalletMagic.jsx

```diff
@@ -67,7 +70,7 @@ const LoadingPrivacyLevel = ({ level }) => {
     <div className="d-flex align-items-center">
       <div className="d-flex">{loadingShields}</div>
       <div className="ps-2">
```

Comment:
Can you can add a width of `4.3rem` to this div, so it matches the non-loading size?

Reply:
Would you tell me what you have in mind? I tried but failed to see how `4.3rem` can be applicable. Any hint would be appreciated!

Balance placeholders are tricky, because the balance can either be shown or hidden, nominated in btc or sats - we can never know what the exact width will be beforehand.

Reply:
Sorry, `4.3rem` only covers one of these cases. I think it's pretty hard to get it correctly for all scenarios, especially because it's centered. Please disregard.